src/cli/input-from-terminal.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const readline = require("readline");
+const { stdin: input } = require("node:process");
+const { printToScreen } = require("./utils.js");
+
+/**
+ * This function starts reading any input which user types on the console.
+ * After the user clicks ctrl+D to exit, the input is formatted and printed
+ * on the screen.
+ * @returns {Promise<string>} On success, resolves the input code joined together
+ * to form a single string. On error, reject with an error message
+ */
+function getInputFromTerminal() {
+  return new Promise((resolve, reject) => {
+    printToScreen("Start writing your code snippet below");
```

Comment:
Let's print the message before calling this function?

Reply:
sounds good to me.